pyvast/README.md

```diff
@@ -35,7 +35,8 @@ and the analogous `pyvast` commands.
   ```
   ```py
   # python wrapper
-  stdout, stderr = vast.export(max_events=10).json("192.167.1.102").exec()
+  proc = await vast.export(max_events=10).json("192.167.1.102").exec()
+  stdout, stderr = await proc.communicate()
```

Comment:
Just for my understanding: from a users perspective, I could imagine to have something like this:
```python
stdout, stderr =  await vast.export(max_events=10).json("192.167.1.102")
```
What is the reason for `.exec()` and the two-step with `.communicate()`?

Reply:
That is the fix that allows to use continuous queries with the PyVAST wrapper. The user is presented the process handler `proc` directly, so the user can poll for new lines in `stdout`.
 
The example you highlighted uses `proc.communicate()` which blocks until EOF in stdout is reached. That was the default before this PR. Now it is up to user to wait (`communicate`) or rather poll `proc.stdout.readline`.

Reply:
Makes sense, thanks.